src/socket/src/sink.rs

```diff
@@ -71,28 +71,22 @@ impl FluvioSink {
     where
         RequestMessage<R>: FlvEncoder + Default + Debug,
     {
-        let bytes = req_msg.as_bytes(0)?;
-        trace!(
-            "sending one way request: {:#?}, bytes: {}",
-            &req_msg,
-            bytes.len()
-        );
-        (&mut self.inner).send(bytes).await?;
+        trace!("sending one way request: {:#?}", &req_msg,);
+        (&mut self.inner).send(req_msg).await?;
         Ok(())
     }
 
     /// as server, send back response
     pub async fn send_response<P>(
         &mut self,
         resp_msg: &ResponseMessage<P>,
-        version: Version,
+        _version: Version,
```

Comment:
This is no longer used with this change. Is that a problem?

Reply:
Version need to be used during encoding and decoding